src/getDependabotPatch.js

```diff
@@ -1,88 +1,87 @@
-export default async function getDependabotPatch({ owner, repo, prnum,
-    githubToken = null,
-    github = null,
-    debug = false
+export default async function getDependabotPatch ({
+  owner, repo, prnum,
+  githubToken = null,
+  github = null,
+  debug = false
 }) {
-    if (!github && githubToken) {
-        const { Octokit } = await import("@octokit/core");
+  if (!github && githubToken) {
+    const { Octokit } = await import('@octokit/core')
 
-        github = new Octokit({ auth: githubToken })
-    }
+    github = new Octokit({ auth: githubToken })
+  }
 
-    if (debug)
-        console.log(`getDependabotPatch ${owner} ${repo} ${prnum}`);
+  if (debug) { console.log(`getDependabotPatch ${owner} ${repo} ${prnum}`) }
 
-    if (!github && !githubToken) {
-        throw new Error("You must provide a githubToken to use this function");
-    }
+  if (!github && !githubToken) {
+    throw new Error('You must provide a githubToken to use this function')
+  }
 
-    prnum = parseInt(prnum, 10);
+  prnum = parseInt(prnum, 10)
 
-    // get PR description
-    const prbodyQuery = `query PRBody($owner:String!, $name:String!, $number:Int!) {
+  // get PR description
+  const prbodyQuery = `query PRBody($owner:String!, $name:String!, $number:Int!) {
         repository(owner: $owner, name: $name) {
           pullRequest(number:$number) {
             body
           }
         }
-    }`;
-    const prbodyVariables = {
-        owner,
-        name: repo,
-        number: prnum
-    };
-    const result = await github.graphql(prbodyQuery, prbodyVariables);
+    }`
+  const prbodyVariables = {
+    owner,
+    name: repo,
+    number: prnum
+  }
+  const result = await github.graphql(prbodyQuery, prbodyVariables)
 
-    // https://github.com/semgrep/semgrep/compare/v1.53.0..v1.54.0.diff
+  // https://github.com/semgrep/semgrep/compare/v1.53.0..v1.54.0.diff
 
-    const firstline = result.repository.pullRequest.body.split("\n")[0];
-    const match = /^Bumps \[[^\]]*\]\(([^\)]*)\) from ([0-9\.]*) to ([0-9\.]*)\./.exec(firstline);
```

Comment:
only need to escape `^-]\` when inside a character class (https://stackoverflow.com/questions/399078/what-special-characters-must-be-escaped-in-regular-expressions)